XamFormsSample/XamFormsSample/XamFormsSample.csproj

```diff
@@ -11,6 +11,11 @@
   </ItemGroup>
   
   <ItemGroup>
+    <None Remove="MainPage.xaml" />
+    <None Remove="App.xaml" />
```

Comment:
Yep, will update to `Remove="**\*.xaml"` which is much better.

I recommend you voice your valuable opinion on the Bugzilla issue, it's likely still up for discussion if we have enough votes for it.